features/youth/conviction_motoring.feature

```diff
@@ -52,7 +40,7 @@ Feature: Youth Conviction
     Examples:
       | subtype                    | endorsement | known_date_header                        | result               | spent_date                                      |
       | Fine                       | Yes         | When were you given the fine?            | /steps/check/results | This conviction will be spent on 1 July 2022    |
-      | Fine                       | No          | When were you given the fine?            | /steps/check/results | This conviction will be spent on 1 July 2020    |
+      | Fine                       | No          | When were you given the fine?            | /steps/check/results | This conviction was spent on 1 July 2020        |
```

Comment:
We need to freeze the time when running the cukes so the calculations are valid no matter when we run the tests.
But that can be done as a separate PR/ticket.

Reply:
https://github.com/ministryofjustice/disclosure-checker/pull/361/commits/184e8bb3c5f8b039a43b5e22744bdcd03d22d0f4